src/status_im/ui/screens/wallet/send/subs.cljs

```diff
@@ -4,6 +4,13 @@
             [status-im.utils.money :as money]
             [status-im.utils.hex :as utils.hex]))
 
+(re-frame/reg-sub :wallet.sent/on-press-got-it
```

Comment:
Comment would be nice: not sure what `on-press-got-it` means.

Reply:
it's better to have a good name for it do you have a better suggestion ? Maybe on-press-got-it-button-event ?